lib/i18n/tasks/translators/deepl_translator.rb

```diff
@@ -112,8 +112,9 @@ def configure_api_key!
     end
 
     def options_with_glossary(options, from, to)
-      glossary = find_glossary(from, to)
-      glossary ? { glossary_id: glossary.id }.merge(options) : options
+      configured = @i18n_tasks.translation_config[:deepl_glossary_ids]
```

Comment:
It would be nice with some documentation for this edge case. Would @glebm prefer it as a code comment or should we add a section to the Readme?

Reply:
Either one is fine but I agree that it'd be good to have this documented

Reply:
@moguls753 Will you please add a comment describing the case?

Reply:
That's a good idea! I added a comment above.